Markdown/Markdown.sublime-syntax

```diff
@@ -1134,6 +1148,7 @@ contexts:
     - match: |-
          (?x)
           {{fenced_code_block_start}}
+          \s*
           ((?i:ruby|rb|rbx))
           \s*$\n?          # any whitespace until EOL
```

Comment:
it may also make sense to move the `\s*$\n?          # any whitespace until EOL` into a variable so we can modify that to get GFM example 112 working with `~~~~    ruby startline=3 $%@#$`.

Reply:
I moved this to a new variable and adapted it to get [GFM example 112](https://github.github.com/gfm/#example-112) working, except for the part where the code block is closed by `~~~~~~~` (i.e., more `~` than the start of this code block `~~~~`). I also added a test for this.